tests/integration/kubernetes/confidential/aa.bat

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../confidential/lib.sh"
+
+# Allow to configure the runtimeClassName on pod configuration.
+RUNTIMECLASS="${RUNTIMECLASS:-kata-qemu}"
+test_tag="[cc][agent][kubernetes][containerd]"
+original_kernel_params=$(get_kernel_params)
+
+setup() {
+    start_date=$(date +"%Y-%m-%d %H:%M:%S")
+
+    kubernetes_delete_all_cc_pods_if_any_exists || true
+
+    echo "Prepare containerd for Confidential Container"
+    SAVED_CONTAINERD_CONF_FILE="/etc/containerd/config.toml.$$"
+#    configure_cc_containerd "$SAVED_CONTAINERD_CONF_FILE"
+
+    echo "Reconfigure Kata Containers"
+    switch_image_service_offload on
+    clear_kernel_params
+    add_kernel_params "${original_kernel_params}"
+
+    setup_proxy
+    switch_measured_rootfs_verity_scheme none
+
+}
+
+@test "$test_tag Test can getevidence" {
+
+    if [ "${AA_KBC}" = "offline_fs_kbc" ]; then
+        setup_offline_fs_kbc_aa_files_in_guest
```

Comment:
Don't need this.

(When we add support for s390x for this test, we may need to do something like this)